index.html

```diff
@@ -49,17 +53,16 @@
         <label for="weeklyMiles">500</label>
         <label class="float-right" for="weeklyMiles">3500</label>
       </div> -->
-      <div class="form-row">
-        <div class="form-label-group col">
-          <input required type="number" pattern="[0-9]*" inputmode="numeric" id="weeklyMiles" name="weeklyMiles" class="form-control" placeholder="Weekly Avg Miles">
-          <label for="weeklyMiles">Weekly Avg Miles</label>
-          <div class="invalid-feedback">How many miles do you run a week?</div>
-        </div>
-        <div class="form-label-group col">
-          <input required type="number" pattern="[0-9]*" inputmode="numeric" step="0.01" id="truckMpg" name="truckMpg" class="form-control" placeholder="Truck MPG">
-          <label for="truckMpg">Truck MPG</label>
-          <div class="invalid-feedback">How many MPG does your truck get?</div>
-        </div>
+      <div class="form-group">
+        <label class="my-1" for="weeklyMiles">Weekly Miles</label>
+        <input required type="tel" id="weeklyMiles" name="weeklyMiles" class="form-control" placeholder="Example: 3,200" data-mask="0,000" data-mask-reverse="true">
+        <div class="invalid-feedback">How many miles do you run a week?</div>
+      </div>
```

Comment:
We lost the ever-present label in favor of the example text label for most (but not all) inputs.